### ISSUE TITLE
[install] Create an advanced dual-boot installer

### ISSUE THREAD

**Issue by LukeShortCloud** [winesapOS/winesapOS#875]
Create an advanced dual-boot installer with a GUI (it could probably be used for regular single-boot installs, too) either using Calamares or another framework.

https://calamares.io/

A simple first iteration of our dual-boot installer will be merged soon from: https://github.com/LukeShortCloud/winesapOS/pull/853/files

**Comment by GuestSneezeOSDev** [comment]
Seems simple enough

**Comment by LukeShortCloud** [comment]
Yes, the goal is to have a GUI dual-boot installer. Ideally with a standardized framework such as Calamares. I have some friends in the community who are using Calamares already. I need to reach out to them for some guidance.

The long-term goal is to replace our existing [dual-boot installer script](https://github.com/winesapOS/winesapOS/blob/4.3.0/rootfs/usr/local/bin/winesapos-dual-boot.sh) with Calamares.

**Comment by LukeShortCloud** [comment]
As far as AUR packages go, [calamares-git](https://aur.archlinux.org/packages/calamares-git) (not [calamares](https://aur.archlinux.org/packages/calamares) which seems to be abandoned) seems to be the go-to package for us since it is maintained by one of the lead Chaotic AUR developers.

**Comment by LukeShortCloud** [comment]
Dual-boot is now out-of-scope. We have an experimental dual-boot installer but we don't even have a regular installer.

The easiest path forward is to use the ChimeraOS installer. There are a few forks of it already.

- [ChimeraOS](https://github.com/ChimeraOS/chimeraos)
    - [KazetaOS](https://github.com/kazetaos/install-media) = Most generic.
    - [Playtron GameOS](https://github.com/playtron-os/install-media)

I talked with @alkazar and the best-case scenario is to merge all of these projects into one instead of us forking the installer yet again. They are mostly the same but they all have different names and scripts for how the installer works.

Another option is to mimic the new GameOS installer by copying our live image, updating the UUIDs of the partitions, and truncating the `/etc/machine-id`.

https://github.com/playtron-os/gameos/blob/new-installer-fixes/rootfs/usr/libexec/playtron/os-install

**Comment by LukeShortCloud** [comment]
I finally got a prototype of Calamares working with Fedora. Untested with macOS, SteamOS, Ubuntu, or Windows.

The only problem I've noticed so far is that the GRUB `os-prober` plugin no longer works with Fedora. That's not a huge deal because all computers with UEFI also have a built-in boot menu.

**Comment by LukeShortCloud** [comment]
The installer has been tested with Fedora 42 and Ubuntu 24.04. Both are working and showing up in our GRUB menu after installation.  I still need to at least test macOS 26, SteamOS 3.8, and Windows 11.

Ideally, I'd like to also remove or consolidate our manual partitioning guide.
